pkg/apis/wksprovider/machine/os/os.go

```diff
@@ -215,19 +217,27 @@ func CreateSeedNodeSetupPlan(ctx context.Context, o *OS, params SeedNodeParams)
 		return nil, err
 	}
 	// Replace w/ info from apply
-	params.AssetDescriptions = map[string]kubeadm.AssetDescription{
-		"DNS": {
-			ImageRepository: "public.ecr.aws/eks-distro/coredns",
-			ImageTag:        "v1.7.0-eks-1-18-1",
-		},
-		"Etcd": {
-			ImageRepository: "public.ecr.aws/eks-distro/etcd-io",
-			ImageTag:        "v3.4.14-eks-1-18-1",
-		},
-		"Kubernetes": {
-			ImageRepository: "public.ecr.aws/eks-distro/kubernetes",
-			ImageTag:        "-eks-1-18-1",
-		},
+	var flavor *eksd.EKSD = nil
+	var err error
+	if params.Flavor.Name == eksd.Flavor {
+		flavor, err = eksd.New(params.Flavor.ManifestURL)
+		if err != nil {
+			return nil, err
+		}
+	}
+	if flavor != nil {
+		params.AssetDescriptions = map[string]kubeadm.AssetDescription{}
+		for _, n := range []string{"Coredns", "Etcd", "Kubernetes"} {
```

Comment:
Super nit... Using "n" as a variable name makes me think of an integer, not a name :man_shrugging: